ts/src/base/Exchange.ts

```diff
@@ -6377,6 +6378,20 @@ export default class Exchange {
         return reconstructedDate;
     }
 
+    async fetchPositionHistory (symbol: Str = undefined, since: Int = undefined, limit: Int = undefined, params = {}): Promise<Position[]> {
```

Comment:
Most endpoints return the "historical" position for multiple symbols, so I think this should be renamed to 
`async fetchPositionsHistory (symbols: Strings = undefined,`